catmux/session.py

```diff
@@ -101,11 +101,30 @@ def _parse_common(self):
         if "common" in self.__yaml_data and self.__yaml_data["common"]:
             common = self.__yaml_data["common"]
             if "before_commands" in common and common["before_commands"]:
-                self._before_commands = common["before_commands"]
+                # self._before_commands = common["before_commands"]
+                self._parse_before_commands(common["before_commands"])
 
             if "default_window" in common and common["default_window"]:
                 self._default_window = common["default_window"]
 
+    def _parse_before_commands(self, commands):
```

Comment:
Why is parsing `before_commands` different to `commands`? Maybe we should unify the concepts? In the end, both should be the same, just parsed from a different section of the config. This would obviously mean that commands would also receive an option to make them conditional, but that's not necessarily a bad thing,

Reply:
Not 100% sure which 'commands' you refer to. But we could parse the 'default_window' command the same way